test/integration/framework/BUILD

```diff
@@ -15,9 +20,14 @@ go_library(
         "test_server.go",
         "util.go",
     ],
-    data = [
-        "@com_coreos_etcd//:etcd",
-    ],
+    data = select({
+        ":cpu_arm64": [
```

Comment:
minor nit: for consistency with other rules, rather than using your own constraint here, use the one defined in `//build:platforms.bzl`:
```python
load("//build:platforms.bzl", "go_platform_constraint")
...
    data = select({
        go_platform_constraint(os="linux", arch="arm64"): ["@com_coreos_etc_arm64//:etcd"],
        "//conditions:default": ...
    }),
```

Reply:
or for extra credit, you could use the `for_platforms` macro in `//build:plaforms.bzl" (assuming you change the name of the amd64 dependency)

```python
load("//build:platforms.bzl", "for_platforms")
...
    data = select(for_platforms(
        for_server=["@com_coreos_etc_{ARCH}//:etcd"],
        only_os="linux",
    }),

Reply:
Done.